web-registry/src/App.tsx

```diff
@@ -182,6 +183,10 @@ const App: React.FC = (): JSX.Element => {
               />
             </Route>
           </Route>
+          <Route
+            path="credit-batch-issuance"
+            element={<ProtectedRoute component={CreditBatchIssuance} />}
```

Comment:
this shouldn't be a protected route

Reply:
Also, similarly to what's described in https://github.com/regen-network/regen-registry/issues/910, I think we should only create this route if the current user is connected to Keplr.

Reply:
:+1 
Adding a not-found page if no `chainId` for now. When 910 is implemented, maybe there will be a message to connect Keplr?

Reply:
I added an error banner to the page to prompt for Keplr. Hiding the route would seem off to me. It makes sense to hide the keplr-related features, but if we send users to a Not-Found page, they wont be prompted to connect a wallet.

Reply:
sounds good, I think replicating the behavior we had for auth0 `ProtectedRoute`s makes sense

Reply:
ideally we could build some `KeplrProtectedRoute` component that could be re-usable for project and credit class creation flows

Reply:
Yeah that's a good idea. The route should be valid for all users, but have a message to connect keplr.

Reply:
I updated the description in https://github.com/regen-network/regen-registry/issues/910
since we're gonna need to do the same for project creation forms, that could be refactored as part of it